src/components/gamePage/GamePage.tsx

```diff
@@ -53,13 +55,37 @@ export const GamePage = ({ socket }: GamePageProps) => {
           {orderedPlayers.map((el, index) => {
             if (index !== 0) {
               return (
-                <div key={el.id} className={style.player}>
-                  <img
-                    className={style.avatar}
-                    src={avatarsArray[+el.avatarId]}
-                    alt="Player avatar"
-                  />
-                  <div className={style.name}>{el.name}</div>
+                <div key={el.id} className={style.playerWrapper}>
+                  <div className={style.playerCards}>
+                    {[...new Array(+el.cards)].map((_, i) => {
+                      const count = +el.cards;
+                      const angle = 100;
+                      const offset = angle / 2;
+                      const increment = angle / (count + 1);
+
+                      return (
+                        <img
+                          key={`${el.id}-${i + 1}`}
+                          className={style.playerCard}
+                          style={{
+                            transform: `translate(-50%, -50%) rotate(${
+                              -offset + increment * (i + 1)
+                            }deg)`,
+                          }}
+                          src={cardback}
+                          alt="Card"
+                        />
+                      );
+                    })}
+                  </div>
+                  <div className={style.player}>
+                    <img
+                      className={style.avatar}
+                      src={avatarsArray[+el.avatarId]}
+                      alt="Player avatar"
+                    />
+                    <div className={style.name}>{el.name}</div>
+                  </div>
                 </div>

```

Comment:
Maxim will 100% want this to be a separate component

Reply:
Done